arangod/Transaction/Manager.h

```diff
@@ -106,11 +107,11 @@ class Manager final {
 
   // register a transaction
   void registerTransaction(TRI_voc_tid_t, std::unique_ptr<TransactionData> data,
-                           bool isReadOnlyTransaction);
+                           bool isReadOnlyTransaction, bool isFollowerTransaction = false);
```

Comment:
IMHO it would be good to not use default values here, but to have the caller explicitly pass a `false` here. That way we don't miss any places that call this function and pass the wrong input value to it.

Reply:
Agreed, and done.